content/en/opentelemetry/compatibility.md

```diff
@@ -0,0 +1,87 @@
+| Universal Service Monitoring (USM)     |                                    |                                    | {{< X >}}                           |                                                | {{< X >}}                          |
+| Cloud SIEM                             | {{< X >}}                          | {{< X >}}                          | {{< X >}}                           | {{< X >}}                                      | {{< X >}}                          |
+| Correlated Traces, Metrics, Logs       | {{< X >}}                          | {{< X >}}                          | {{< X >}}                           | {{< X >}}                                      | {{< X >}}                          |
+| Live Container Monitoring/K8s Explorer |                                    | {{< X >}}                          | {{< X >}}                           | {{< X >}}                                      | {{< X >}}                          |
+| Network Performance Monitoring (NPM)   |                                    | {{< X >}}                          | {{< X >}}                           | {{< X >}}                                      | {{< X >}}                          |
+| Real User Monitoring (RUM)             | {{< X >}}                          | {{< X >}}                          | {{< X >}}                           | {{< X >}}                                      | {{< X >}}                          |
+| Runtime Metrics                        | {{< X >}}<br>(Java, .NET, Go only) | {{< X >}}<br>(Java, .NET, Go only) | {{< X >}}                           | {{< X >}}                                      | {{< X >}}<br>(Java, .NET, Go only) |
+| Source code integration                | {{< X >}}                          | {{< X >}}                          | {{< X >}}                           |                                                | {{< X >}}                          |
+
+## More details
+
+### Runtime Metrics
+
+Setups using the OpenTelemetry SDK follow the [OpenTelemetry Runtime Metrics][1] specification.
+
+### Real User Monitoring (RUM)
+
+To enable full RUM functionality, you need to [inject supported headers][2] to correlate RUM and traces.
+
+### Network Performance Monitoring (NPM)
+
+For **OTel to Datadog (OTLP)** and **OTel with embedded Datadog Collector (Preview)** setups, span-level or endpoint-level monitoring is **not** supported.
+
+For more information, see [Network Performance Monitoring Setup][3].
+
+### Live Processes
+
+For **OTel to Datadog (OTLP)**, you must enable the [Process Agent][4].
+
+### Source Code Integration
+
+For unsupported languages in OpenTelemetry setups, [configure telemetry tagging][5] to link data to a specific commit.
+
+## Best practices
+
+- **Avoid Mixed Instrumentation**: Don't use both Datadog SDK and OpenTelemetry SDK to instrument the same application.
```

Comment:
Avoid Mixed Instrumentation: We recommend against using both a Datadog SDK and OpenTelemetry SDK to instrument the same application. This leads to undefined behaviors.

Avoid Agent and Separate Collector on Same Host: We recommend against using the Datadog Agent and a separate OpenTelemetry Collector at the same time on the same host, as this may cause problems. Using Agents and Collectors on different hosts in the same fleet is, however, supported.